Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN wget -P /tmp https://github.com/microsoft/ApplicationInsights-Java/releases/
 
 # Application image
 
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.4
+FROM hmctspublic.azurecr.io/base/java:17-distroless
 
 COPY --from=downloader /tmp/applicationinsights-agent-${APP_INSIGHTS_AGENT_VERSION}.jar /opt/app/
```

Comment:
need custom applicationinsights.json
no need to copy jar.

Reply:
you can delete lib/ai.xml

Reply:
ARG APP_INSIGHTS_AGENT_VERSION should be 3.X
sth :
ARG APP_INSIGHTS_AGENT_VERSION=3.2.6